packages/web/src/app/doc-pages/components/toast-doc/toast-data.ts

```diff
@@ -6,5 +6,53 @@ import ComponentData from '../component-data.interface';
 export const toastData: ComponentData<ToastProps> = {
   changelog: changelogJson.content,
   name: 'Toast',
-  attributes: {},
+  attributes: {
+    openElviaToast: {
+      isRequired: false,
+      type: '(params: ToastConfigType) => void',
+      specialType: 'function',
+      children: {
+        title: {
+          isRequired: false,
+          type: 'string',
+          description: 'The title is shown in the toast.',
+          specialType: 'function',
```

Comment:
I haven't thoroughly checked the underlying code, but is it necessary that all the `children` of a `specialType: 'function'` are also marked as `specialType: 'function'`, or could that be inferred somehow so that it doesn't need to be repeated so many times?